scripts/node-subsystem-label.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const debug = require('debug')('node_subsystem_label')
+
+const nodeRepo = require('../lib/node-repo')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', handlePrCreated)
+
+  function handlePrCreated (event) {
```

Comment:
This signature is now `(event, org, repo, sender)` if you want to use them and remove the next few lines.